spidermon/contrib/actions/email/ses.py

```diff
@@ -1,4 +1,3 @@
-from __future__ import absolute_import
 from boto import ses
```

Comment:
GitHub shows something weird above this line for me (a line `0`, and a duplicate `session.send_raw_email(raw_message=message.as_string())` line being turned into a single one in this patch), but I checked `master` and the line before there was `from __future__ import absolute_import`, so :heavy_check_mark: